runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -426,17 +426,26 @@ public String getXWalkVersion() {
         return nativeGetVersion(mNativeContent);
     }
 
+    private boolean isOpaque(int color) {
+        return ((color >> 24) & 0xFF) == 0xFF;
+    }
+
     public void setBackgroundColor(final int color) {
         if (mNativeContent == 0) return;
         if (mIsLoaded == false) {
             post(new Runnable() {
                 @Override
                 public void run() {
-                    nativeSetBackgroundColor(mNativeContent, color);
+                    setBackgroundColor(color);
                 }
             });
             return;
         }
+        if (isOpaque(color) == false) {
+            setOverlayVideoMode(true);
+            mContentViewRenderView.setSurfaceViewBackgroundColor(color);
+            mContentViewCore.setBackgroundOpaque(false);
+        }
```

Comment:
No need. This is specially for transparent background. For normal color, no need.